library/koji_tag.py

```diff
@@ -177,7 +181,14 @@ def ensure_inheritance(session, tag_name, tag_id, check_mode, inheritance):
         rules.append(new_rule)
     current_inheritance = session.getInheritanceData(tag_name)
     if current_inheritance != rules:
-        result['stdout_lines'].append('inheritance is %s' % inheritance)
+        result['stdout_lines'].extend(
+                ('current inheritance:',)
+                + tuple(map(lambda i: "%(priority)4d   .... %(name)s" % i,
+                            current_inheritance))
+                + ('would set inheritance:' if check_mode
+                    else 'new inheritance:',)
+                + tuple(map(lambda i: "%(priority)4d   .... %(name)s" % i,
+                            rules)))
```

Comment:
This is a great feature. Maybe we should refactor into a separate `describe_inheritance()` method that returns the string (or list of strings) we need. That would also buy us more indentation space as well here. What do you think?

Reply:
Good idea; done.